CHANGELOG.md

```diff
@@ -32,6 +33,7 @@ These are the section headers that we use:
 - Added Telemetry support for `ArgillaTrainer` (closes [#3325](https://github.com/argilla-io/argilla/issues/3325))
 - `User.workspaces` is no longer an attribute but a property, and is calling `list_user_workspaces` to list all the workspace names for a given user ID ([#3334](https://github.com/argilla-io/argilla/pull/3334))
 - Renamed `FeedbackDatasetConfig` to `DatasetConfig` and export/import from YAML as default instead of JSON (just used internally on `push_to_huggingface` and `from_huggingface` methods of `FeedbackDataset`) ([#3326](https://github.com/argilla-io/argilla/pull/3326)).
+- Renamed `/client/feedback/typing.py` to `/client/feedback/types.py` to avoid import errrors.
```

Comment:
IMO there's no need to add this into the `CHANGELOG.md` as that's an internal change, nothing that affects the users, otherwise, in case we keep it, we should specify the complete path to avoid issues i.e.
```suggestion
- Renamed `src/argilla/client/feedback/typing.py` to `src/argilla/client/feedback/types.py` to avoid import errors.
```